src/discord/commands/ranklist.ts

```diff
@@ -0,0 +1,120 @@
+import { SlashCommandBuilder } from "discord.js";
+import { Command } from "../type";
+import { db } from "$db/index";
+import axios from "axios";
+import CliTable3 from "cli-table3";
+
+export const ranklistCmd: Command = {
+    info: new SlashCommandBuilder()
+        .setName("ranklist")
+        .setDescription("Get the ranklist for a specific contest")
+        .addIntegerOption(option => 
+            option.setName("contest")
+                .setDescription("The contest number")
+                .setRequired(true)
+        ),
+
+    async execute(msg) {
+        const contestId = msg.options.getInteger("contest");
+
+        // Fetch user handles from the database
+        const userHandles = await fetchUserHandlesFromDB();
+
+        // Convert the set of user handles to a semicolon-separated string
+        const handlesParam = Array.from(userHandles).join(';');
+
+        // Make an API request to contest.standings with the handles parameter
+        const url = `https://codeforces.com/api/contest.standings?contestId=${contestId}&from=1&count=50000&showUnofficial=true&handles=${handlesParam}`;
```

Comment:
No direct calls to API must be made, make a call via the cf api class, you can refer the other commands on how it works - if your command requires a new api request that's not defined - you can just add that function in the class